docker/hvd/Dockerfile.hvd-apex

```diff
@@ -25,9 +25,12 @@ ARG HVD_VERSION
 
 # Build Horovod
 RUN apt-get update && apt-get install -y git && \
-        git clone --recursive --depth 1 --branch ${HVD_VERSION} https://github.com/horovod/horovod.git /horovod && \
-        conda install -y cmake nccl -c conda-forge && \
+        conda install -y cmake=3.* nccl scikit-build -c conda-forge && \
```

Comment:
I'm surprised a bit seeing scikit-build as dependency, do we really need it?

Reply:
I was seeing a build error pecause that was missing on my initial tests, but it was probably to the branch no being the right one at checkout. 

Will revert!